requests/test/src/requests/RequestTests.scala

```diff
@@ -99,35 +113,38 @@ object RequestTests extends TestSuite{
         assert(read(res2) == Obj("cookies" -> Obj("freeform" -> "test test", "hello" -> "hello, world")))
       }
     }
-    // Tests fail with 'Request to https://httpbin.org/absolute-redirect/4 failed with status code 404'
-    // test("redirects"){
-      // test("max"){
-      //   val res1 = requests.get("https://httpbin.org/absolute-redirect/4")
-      //   assert(res1.statusCode == 200)
-      //   val res2 = requests.get("https://httpbin.org/absolute-redirect/5")
-      //   assert(res2.statusCode == 200)
-      //   val res3 = requests.get("https://httpbin.org/absolute-redirect/6", check = false)
-      //   assert(res3.statusCode == 302)
-      //   val res4 = requests.get("https://httpbin.org/absolute-redirect/6", maxRedirects = 10)
-      //   assert(res4.statusCode == 200)
-      // }
-      // test("maxRelative"){
-      //   val res1 = requests.get("https://httpbin.org/relative-redirect/4")
-      //   assert(res1.statusCode == 200)
-      //   val res2 = requests.get("https://httpbin.org/relative-redirect/5")
-      //   assert(res2.statusCode == 200)
-      //   val res3 = requests.get("https://httpbin.org/relative-redirect/6", check = false)
-      //   assert(res3.statusCode == 302)
-      //   val res4 = requests.get("https://httpbin.org/relative-redirect/6", maxRedirects = 10)
-      //   assert(res4.statusCode == 200)
-      // }
-    // }
+
+    //Tests fail with 'Request to https://httpbin.org/absolute-redirect/4 failed with status code 404'
```

Comment:
Did you fix those tests? If so you should remove this comment, right?

Reply:
I'd buy that